src/main/java/emissary/util/shell/Executrix.java

```diff
@@ -973,7 +974,7 @@ public String[] getCommand(final String commandArg, final String[] tmpNames, fin
         c = c.replaceAll("<OUTPUT_NAME>", tmpNames[OUT]);
 
         final String[] cmd;
-        if (System.getProperty("os.name").startsWith("Windows")) {
+        if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
I thought _(hoped?)_ we were dropping Windows support?  (I'm sure there's a separate PR that addresses that)

Reply:
https://github.com/NationalSecurityAgency/emissary/pull/255